python_modules/libraries/dagster-aws/dagster_aws/ecs/tasks.py

```diff
@@ -125,6 +130,8 @@ def from_task_definition_dict(task_definition_dict, container_name):
             task_role_arn=task_definition_dict.get("taskRoleArn"),
             sidecars=sidecars,
             requires_compatibilities=task_definition_dict.get("requiresCompatibilities"),
+            cpu=task_definition_dict.get("cpu"),
```

Comment:
This will still be overridable via run tags - that feels like the correct behavior to me, but I wanted to call it out to make sure you agree.